docs/content/latest/architecture/transactions/read-committed.md

```diff
@@ -0,0 +1,1205 @@
+```
+
+```
+UPDATE 1
+```
+   </td>
+  </tr>
+  <tr>
+   <td>
+
+```sql
+update test set v=100 where v>=5;
+```
+
+```
+(waits)
+```
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>
+
+```sql
+commit;
+```
+   </td>
+  </tr>
+  <tr>
+   <td>
+
+```
+UPDATE 4
+```
+
+```sql
+select * from test;
+```
+
+```
+ k  |  v
+----+-----
+  5 | 100
+  1 |   1
+ 10 | 100
+  4 | 100
+  2 | 100
+(5 rows)
+```
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>
+
+```sql   
+commit;
+```
+
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+
+### 4) Behavior of SELECT FOR UPDATE
+
+
+```sql
+truncate table test;
+insert into test values (0, 5), (1, 5), (2, 5), (3, 5), (4, 1);
+```
+
+<table>
+  <tr>
+   <td>
+   Client 1
+   </td>
+   <td>
+   Client 2
+   </td>
+  </tr>
+  <tr>
+   <td>
+
+```sql
+begin transaction isolation level read committed;
+```
+
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>
+   
+```sql
+begin transaction isolation level read committed;
+```
+
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>
+
+```sql
+insert into test values (5, 5);
+```
+
+```
+INSERT 0 1
+```
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>
+
+```sql
+update test set v=10 where k=4;
+```
+
+```
+UPDATE 1
+```
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>
+
+```sql
+delete from test where k=3;
+```
+
+```
+DELETE 1
+```
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>
+
+```sql
+update test set v=10 where k=2;
+```
+
+```
+UPDATE 1
+```
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>
+
+```sql
+update test set v=1 where k=1;
+```
+
+```
+UPDATE 1
+```
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>
+
+```sql
+update test set k=10 where k=0;
+```
+
+```
+UPDATE 1
+```
+   </td>
+  </tr>
+  <tr>
+   <td>
+
+```sql
+select * from test where v>=5 for update;
+```
+
+```
+(waits)
+```
+
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>
+
+```sql
+commit;
+```
+   </td>
+  </tr>
+  <tr>
+   <td>
+   
+```
+ k  | v
+----+----
+  5 |  5
+ 10 |  5
+  4 | 10
+  2 | 10
+(4 rows)
+```
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>
+
+```sql
+commit;
+```
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+
+
+### 5) Behavior of INSERTs
+
+
+**i) insert new key that is also just added/ changed by another transaction**
+
+
+```sql
+truncate table test;
+insert into test values (1, 1);
+```
+
+<table>
+  <tr>
+   <td>
+   Client 1
+   </td>
+   <td>
+   Client 2
+   </td>
+  </tr>
+  <tr>
+   <td>
+
+```sql
+begin transaction isolation level read committed;
+```
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>
+
+```sql
+begin transaction isolation level read committed;
+```
+
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>
+
+```sql
+update test set k=2 where k=1;
+```
+
+```
+UPDATE 1
+```
+   </td>
+  </tr>
+  <tr>
+   <td>
+
+```sql
+insert into test values (2, 1); -- insert new key that is added by other transaction
```

Comment:
can you put the comment on a separate line, that way the break will be better and the table will display less lopsided

Reply:
done